azext_edge/edge/params.py

```diff
@@ -969,70 +990,121 @@ def load_iotops_arguments(self, _):
+            type=int,
+            arg_group="Connector",
         )
         context.argument(
-            "tags",
-            options_list=["--tags"],
-            help="Asset Endpoint resource tags. Property bag in key-value pairs with the following format: a=b c=d",
-            arg_type=tags_type,
+            "keep_alive",
+            options_list=["--keep-alive", "--ka"],
+            help="Time in milliseconds after which a keep alive publish response is sent. Minimum: 0. "
+            "Recommended: 10000.",
+            type=int,
+            arg_group="Connector",
+        )
+        context.argument(  # TODO: change to disable
+            "run_asset_discovery",
+            options_list=["--run-asset-discovery", "--rad"],
+            help="Flag to determine if asset discovery should be run.",
+            arg_type=get_three_state_flag(),
+            arg_group="Connector",
         )
-
-    with self.argument_context("iot ops asset endpoint certificate") as context:
         context.argument(
-            "asset_endpoint_profile_name",
-            options_list=["--endpoint"],
-            help="Asset Endpoint name.",
+            "session_timeout",
+            options_list=["--session-timeout", "--st"],
+            help="Session timeout in milliseconds. Minimum: 0. Recommended: 60000.",
+            type=int,
+            arg_group="Connector",
         )
         context.argument(
-            "password_reference",
-            options_list=["--password-ref", "--pr"],
-            help="Reference for pem file that contains the certificate password.",
-            arg_group=None,
+            "session_keep_alive",
+            options_list=["--session-keep-alive", "--ska"],
+            help="Time in milliseconds after which a session keep alive challenge is sent to detect "
+            "connection issues. Minimum: 0. Recommended: 10000.",
+            type=int,
+            arg_group="Connector",
         )
         context.argument(
-            "secret_reference",
-            options_list=["--secret-ref", "--sr"],
-            help="Reference for the der file that contains the certificate. The referenced file should contain the "
-            "certificate and the key.",
+            "session_reconnect_period",
+            options_list=["--session-reconnect-period", "--srp"],
+            help="Session reconnect period in milliseconds. Minimum: 0. Recommended: 2000.",
+            type=int,
+            arg_group="Connector",
         )
         context.argument(
-            "thumbprint",
-            options_list=["--thumbprint", "-t"],
-            help="Certificate thumbprint.",
+            "session_reconnect_exponential_back_off",
+            options_list=["--session-reconnect-backoff", "--srb"],
+            help="Session reconnect exponential back off in milliseconds. Minimum: -1. Recommended: 10000.",
+            type=int,
+            arg_group="Connector",
+        )
```

Comment:
At some point in the near future, we should organize params (and other command structural elements) into their domain specific modules. This module is getting too long.

Reply:
very yes - tbh I would not mind starting with this pr